net/lp_rpc.proto

```diff
@@ -340,6 +340,9 @@ message NotifySegment {
     // ID for this particular transcoding task.
     int64 taskId   = 16;
 
+    // Orchestrator identifier for segment metadata
+    string orchId  = 18;
```

Comment:
nit: shouldn't you move it under profiles, so that the indexing is in the order?

Reply:
Isn't `profiles` deprecated?